### DISCUSSION TITLE
Compression artifacts with fast moving objects.

### DISCUSSION THREAD

**Discussion by usamanetarus** [AxisCommunications/discussions#85] [General] [answered]
Hi. Can someone please let me know if there's an option to eliminate these compression artifact. 
![image (1)](https://user-images.githubusercontent.com/93617659/196760927-62af90c6-11d1-46b8-a0f4-aec2e807c188.png)

**Comment by haichaoji123** [comment, marked as answer]
try to turn off WDR and use a higher shutter speed.
